Src/Support/Google.Apis.Auth/OAuth2/IGoogleCredential.cs

```diff
@@ -55,6 +55,19 @@ internal interface IGoogleCredential : ICredential, ITokenAccessWithHeaders
         /// </summary>
         bool SupportsExplicitScopes { get; }
 
+        /// <summary>
+        /// Returns true if this credential supports explicit impersonation.
```

Comment:
I'm a bit confused about the terms "implicit" and "explicit impersonation" -- I've never encountered the terms elsewhere, and I'm not sure what they mean.

Isn't the IExternalAccountCredential sufficient to "mark" a credential as being a workload identity federation credential?

I'd rather see us rely on polymorphism than use "SupportsThisAndThat" properties.